scripts/transactions/mass-ops.ts

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { writeFileSync } from 'fs';
+import { TransactionBlock } from '@mysten/sui.js/transactions';
+
+import { mainPackage } from '../config/constants';
+import { removeDiscountForType } from '../config/discounts';
+import { newCouponRules, optionalRangeConstructor, PercentageOffCoupon } from '../coupons/coupon';
+import reservedObjects from '../reserved-names/owned-objects.json';
+import { prepareMultisigTx } from '../utils/utils';
+
+function generateRandomString(length: number) {
+	const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+	let result = '';
+	const charactersLength = characters.length;
+
+	for (let i = 0; i < length; i++) {
+		result += characters.charAt(Math.floor(Math.random() * charactersLength));
+	}
+
+	return result;
+}
+
+const run = async () => {
+	const config = mainPackage.mainnet;
+
+	// This is a list of free names and their addresses.
+	const freeNamesAddresses = [
+		...new Set([
+			'0x29aa83013e8e5646f479188761e510fd4e7062123d0e1f0ffc654ecfde4d7550',
+			'0x74bd991f694eb5792c10c3715a125629da67a0494e95b299ae23264210a53cf5',
+			'0x29aa83013e8e5646f479188761e510fd4e7062123d0e1f0ffc654ecfde4d7550',
```

Comment:
this is listed twice in the request and here (didn't notice until i started verifying addresses.)

Reply:
`0x74bd991f694eb5792c10c3715a125629da67a0494e95b299ae23264210a53cf5`